.github/workflows/dependabot-automerge.yml

```diff
@@ -21,13 +21,15 @@ jobs:
         id: metadata
         uses: dependabot/fetch-metadata@v2
         with:
-          github-token: "${{ secrets.GITHUB_TOKEN }}"
+          github-token: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Ask Dependabot to rebase (if needed)
         if: steps.metadata.outputs.has-conflicts == 'true'
-        uses: bbeesley/gha-auto-dependabot-rebase@v1
+        uses: bbeesley/gha-auto-dependabot-rebase@main
```

Comment:
It's better to pin it to a specific version rather than `main`

Reply:
We could. Suggested workflow on the GitHub page seems to indicate pinning to `main` is ideal:
https://github.com/bbeesley/gha-auto-dependabot-rebase

I've pinned it now to the most recent version (bbeesley/gha-auto-dependabot-rebase@v1.5.129).